src/main/scala-2.12/org/apache/james/gatling/jmap/scenari/JmapInboxHomeLoadingScenario.scala

```diff
@@ -34,23 +39,42 @@ class JmapInboxHomeLoadingScenario {
 
   val listMessagesInInbox: HttpRequestBuilder =
     JmapAuthentication.authenticatedQuery("listMessagesInInbox", "/jmap")
-      .body(StringBody("""[["getMessageList",{"filter":{"inMailboxes":["${inboxID}"],"text":null},"sort":["date desc"],"collapseThreads":false,"fetchMessages":false,"position":0,"limit":200,"#0"]]"""))
+      .body(StringBody(
+        """[["getMessageList",
+          |{"filter":{"inMailboxes":["${inboxID}"],
+          |"text":null},
+          |"sort":["date desc"],
+          |"collapseThreads":false,
+          |"fetchMessages":false,
+          |"position":0,
+          |"limit":200},"#0"]]""".stripMargin))
       .check(jsonPath("$[0][1].messageIds[*]").findAll.saveAs(Keys.messageIds))
 
 
-  private val getMessages: HttpRequestBuilder =     JmapAuthentication.authenticatedQuery("getMessages", "/jmap")
-    .body(StringBody("""[["getMessages",{"properties": ["id","blobId","threadId","headers","subject","from","to","cc","bcc","replyTo","preview","date","isUnread","isFlagged","isDraft","hasAttachment","mailboxIds","isAnswered","isForwarded"],"ids": ${messageIds},"#0"]]"""))
-
-  def generate(feederBuilder: FeederBuilder): ScenarioBuilder = {
+  private val getMessagesDetails: HttpRequestBuilder = JmapAuthentication.authenticatedQuery("getMessages", "/jmap")
```

Comment:
I think it makes sense to use the jmap wording and keep getMessages, don't you think?